pyproject.toml

```
@@ -0,0 +1,112 @@
+[tool.poetry]
+name = "epyqlib"
+version = "0.0.0"
+description = ""
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
+include = ["epyqlib/scripting.csv", "epyqlib/icon.ico", "epyqlib/**/*.ui"]
+license = "GPL-2.0-or-later"
+
+[tool.poetry-dynamic-versioning]
+#modifies epyqlib/__init__.py
+enable = false
+vcs = "git"
+style = "pep440"
+
+[tool.poetry.dependencies]
+python = "^3.7.9"
+
+#### EPC Repos ####
+sunspecdemo = {git = "https://github.com/epcpower/sunspec-demo", rev = "def34227c0bc205083ddffe429c640509ab0bf52"}
+###################
+
+alqtendpy = "0.0.4"
+arrow = "0.12.1"
+appdirs = "1.4.3"
+attrs = ">=19.3.0"
+bitstruct = "6.0.0"
+boto3 = "1.14.41.0"
+boto3-type-annotations = "0.3.1"
+canmatrix = "0.9.1"
+certifi = "2020.6.20"
+certitude = { version = "1.0.1", markers = "sys_platform == 'Darwin'" }
+click = "<=7.1.2"
+dulwich = "0.20.6"
+fab = "3.0.0"
+GitPython = "2.1.15"
+graham = "0.1.11"
+marshmallow = "2.16.3"
+natsort = "5.5.0"
+paho-mqtt = "1.4.0"
+Pint = "0.11"
+pyelftools = "0.26"
+PyQt5 = ">=5.13.0"
+pyqt5-tools = { version = "5.13.2.1.6rc1", markers = "sys_platform == 'win32'" }
+qt5reactor = "0.5"
+python-dateutil = "2.7.5"
+python-docx = "0.8.7"
+python-dotenv = "0.9.1"
+QtAwesome = "0.6.0"
+siphash-cffi = "^0.1.4"
```

Should be locked at v0.1.4. Also need to switch from fstab to AWS CodeArtifact for hosting.
```suggestion
siphash-cffi = "0.1.4"
```

---

Adding now.  not committing suggestion bc lock file needs to be update as well

---

This is still specified to look for versions `^0.1.4`. Should be pinned at `0.1.4`.